src/components/breedcard.css

```diff
@@ -0,0 +1,58 @@
+.breed-card {
+  cursor: pointer;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #548dfe;
+  padding: 0 5px 0 0;
+  box-shadow: 2px 2px 4px 0 rgb(0 0 0 / 20%);
+  color: #fff;
+  margin-bottom: 5px;
+}
+
+.image-part {
+  display: flex;
+  justify-content: space-between;
+}
+
+.breed-card img {
+  width: 90%;
+  height: 25vh;
+}
+
+.text-part {
+  display: flex;
+  flex-direction: column;
+  text-align: end;
+  margin-right: 10px;
+  align-self: end;
+}
```

Comment:
Kindly, match the design provided and here you can use `align-items: flex-end;` for the texts to align at the end and also give padding bottom to give space between individual cards it's kind of collapsed now.